src/main/wrapper/ExecutionService.ts

```diff
@@ -35,12 +36,24 @@ function isJsonString(s: string) {
     return true;
 }
 
-function transformation(commands: string[]):string[] {
-    const result:string[] = commands.map(transform);
+function transformation(commands: string[]): string[] {
+    skipNext = false; // Reset the flag before processing
+    const result: string[] = commands.map(transform);
     return result;
 }
 
 function transform(n:string) {
+    // If the flag is set, return the current string as-is and reset the flag
+    if (skipNext) {
+        skipNext = false;
+        return n;
+    }
+    // If the current string is "--file-source", set the flag and return it as-is
+    if (n === "--file-source") {
```

Comment:
can you use a common const for - "--file-source" ?

Reply:
done